android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -232,9 +252,26 @@ public void onMeetingStatusChanged(MeetingStatus meetingStatus, int errorCode, i
     } else if (meetingStatus == MeetingStatus.MEETING_STATUS_INMEETING) {
       meetingPromise.resolve("Connected to zoom meeting");
       meetingPromise = null;
+
+      connectAudioWithVoIP();
```

Comment:
auto-connect should optional and possible to switch on/off in `ZoomUs.initialize(`